source/comparisons.rst

```diff
@@ -182,6 +193,9 @@ we recommend adding the full, documented function to your project's utility
 library.
 
 
+.. index:: cmp; argument of sort()
+.. index:: TypeError; key function
```

Comment:
Here I think it's unnecessary: "key function" comes from the error message and I don't think it's relevant on its own.